lib/challenge.js

```diff
@@ -114,7 +126,11 @@ function prompt6() {
  *   - I found a 10. High five!
  */
 function prompt7() {
-  // YOUR CODE HERE
+  for(let i = 0; i <= 100; i++) {
+    if(i%5 == 0) {
+      console.log("- I found a " + i + ". High five!");
+    }
+  }
 }
```

Comment:
Check the spacing inside your if statement on line 130.
I'd space out the `i%5` to make it more readable

Also, this may be a preference thing but I like to use the strict `===` comparison operators when I can just to keep my data types and flow straight.  But sometimes it may be useful to have a quick way to compare a string to a number on the fly for example.  Just food for thought